lib/browser/api/dialog.js

```diff
@@ -283,21 +200,24 @@ module.exports = {
     return binding.showErrorBox(...args)
   },
 
-  showCertificateTrustDialog: function (...args) {
-    let [window, options, callback] = parseArgs(...args)
+  showCertificateTrustDialog: (opts) => {
+    if (opts == null) throw new Error('Insufficient number of arguments')
 
-    if (options == null || typeof options !== 'object') {
+    // destructure params with default options value
+    let {window, options = {}, callback} = opts
+
+    if (typeof options !== 'object') {
```

Comment:
Check for `null`?

Reply:
if it's null it'll be set to `{}` per my default setting above, no?

Reply:
No, default values are only used if no value or `undefined` is passes to a function call. So if you'd call `showCertificateTrustDialog({window: <...>, options: null})`, `null` won't be replaced by en empty object.